bin/elixir.ps1

```diff
@@ -0,0 +1,379 @@
+  --eval "COMMAND"             Evaluates the given command, same as -e (*)
+  --logger-otp-reports BOOL    Enables or disables OTP reporting
+  --logger-sasl-reports BOOL   Enables or disables SASL reporting
+  --no-halt                    Does not halt the Erlang VM after execution
+  --short-version              Prints Elixir version (standalone)
+  --werl                       Uses Erlang's Windows shell GUI (Windows only)
+
+Options given after the .exs file or -- are passed down to the executed code.
+Options can be passed to the Erlang runtime using \$ELIXIR_ERL_OPTIONS or --erl.
+
+## Distribution options
+
+The following options are related to node distribution.
+
+  --cookie COOKIE              Sets a cookie for this distributed node
+  --hidden                     Makes a hidden node
+  --name NAME                  Makes and assigns a name to the distributed node
+  --rpc-eval NODE "COMMAND"    Evaluates the given command on the given remote node (*)
+  --sname NAME                 Makes and assigns a short name to the distributed node
+
+--name and --sname may be set to undefined so one is automatically generated.
+
+## Release options
+
+The following options are generally used under releases.
+
+  --boot "FILE"                Uses the given FILE.boot to start the system
+  --boot-var VAR "VALUE"       Makes \$VAR available as VALUE to FILE.boot (*)
+  --erl-config "FILE"          Loads configuration in FILE.config written in Erlang (*)
+  --pipe-to "PIPEDIR" "LOGDIR" Starts the Erlang VM as a named PIPEDIR and LOGDIR
+  --vm-args "FILE"             Passes the contents in file as arguments to the VM
+
+--pipe-to starts Elixir detached from console (Unix-like only).
+It will attempt to create PIPEDIR and LOGDIR if they don't exist.
+See run_erl to learn more. To reattach, run: to_erl PIPEDIR.
+
+--pipe-to is not supported on Windows. If set, Elixir won't boot.
+
+** Options marked with (*) can be given more than once.
+** Standalone options can't be combined with other options.
+"@
+
+  Write-Host $Help
+}
+
+if (($Args.Count -eq 1) -and ($Args[0] -eq "--short-version")) {
+  Write-Host "$ELIXIR_VERSION"
+  exit
+}
+
+if (($Args.Count -eq 0) -or (($Args.Count -eq 1) -and ($Args[0] -in @("-h", "--help")))) {
+  PrintElixirHelp
+  exit 1
+}
+
+function NormalizeArg {
+  param(
+    [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+    [string[]] $Items
+  )
+  [string]::Join(",", $Items)
+}
+
+function QuotedString {
+  param(
+    [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+    [string] $Item
+  )
+
+  # We surround the string with double quotes, in order to preserve its contents as
+  # only one command arg.
+  # This is needed because PowerShell will consider a new argument when it sees a
+  # space char.
+  if (-not $Item.StartsWith("`"") -and $Item.Contains(" ")) {
+    "`"$Item`""
+  }
+  else {
+    $Item
+  }
+}
+
+$ElixirParams = New-Object Collections.Generic.List[String]
```

Comment:
Inconsistent array declaration.  In elixirc.ps1 you're using `@()`, which I suggest you use everywhere.

Reply:
```
$x = @()

# append
$x = $x + 1
# or
$x = $x + @(1, 2, 3)

# prepend
$x = @(2) + $x
```

Either way is fine and most of the code uses the list style so maybe go with that.  I thought it might be nice to be consistent across the scripts.

Reply:
I changed to the `+=` style. Thanks for the suggestion!